deps.edn

```diff
@@ -6,7 +6,8 @@
  :paths   ["src"]
 
  :aliases {:test {:extra-paths ["test"]
-                  :extra-deps  {io.github.cognitect-labs/test-runner {:git/tag "v0.5.0" :git/sha "b3fd0d2"}}
+                  :extra-deps  {io.github.cognitect-labs/test-runner {:git/tag "v0.5.0" :git/sha "b3fd0d2"}
+                                expectations/clojure-test {:mvn/version "1.2.1"}}
```

Comment:
Why the dependency on expectations? As far as I can see, it's not being used.

Reply:
Most keys are prefixed with `::`. 
`::url` won't do it because we want to differentiate between redis/rabbitmq. 
`::redis/url` doesn't work.
Most keys are `::broker-opts`, i.e. `:goose.specs/broker-opts`
Should I replace all `:redis/url` with `:goose.specs.redis/url` for consistency?

Reply:
@jysandy Removing the dependency. Either I'll create a macro, or get the exception as part of a try-catch fn.